images/fastchat-t5-3b/Dockerfile

```diff
@@ -63,13 +60,15 @@ RUN pip install -r requirements.txt
 
 RUN mkdir -p /workspace
 
-COPY main.py workspace/
+COPY . workspace/
+RUN pip install -e .
 
 WORKDIR /workspace
 
-RUN python main.py --dry-run
+RUN modelz-llm --dry-run
```

Comment:
How to cache the model into the  image if we do not specify the env var https://github.com/tensorchord/modelz-llm/pull/37/files#diff-67c9c57206434ccae55c20ae91b640becb6ebab4e299359fd54b0961e495222aL8